src/pub.c

```diff
@@ -290,6 +290,26 @@ const char* DPS_PublicationGetTopic(const DPS_Publication* pub, size_t index)
     }
 }
 
+const DPS_KeyId* DPS_PublicationGetId(const DPS_Publication* pub)
```

Comment:
I think this function name is confusing because the publication id previously meant the publication UUID whereas this is the key id. I suggest renaming this to DPS_PublicationGetKeyId